code/controllers/subsystem/SSsecurity_level.dm

```diff
@@ -0,0 +1,150 @@
+#define DEFAULT_SECURITY_LEVEL_NUMBER SEC_LEVEL_GREEN
+#define DEFAULT_SECURITY_LEVEL_NAME "green"
```

Comment:
Maybe default it to the datum ref and read these properties from that?

Reply:
Not quite understand how to do what you propose